boards/cats_rev1Pro/src/cli/settings.h

```diff
@@ -23,8 +23,7 @@
 
 #define ARRAYLEN(x) (sizeof(x) / sizeof((x)[0]))
 
-typedef enum { TABLE_BOOTSTATE = 0,
-               TABLE_EVENTS,
+typedef enum { TABLE_EVENTS = 0,
```

Comment:
should we have an invalid for 0?

Reply:
This enum is used to index the lookup_tables array; if we change the index to start from 1, we will either have an empty entry in that array, or we have to shift the index by ENUM - 1 every time, which seems more error-prone